django_filters/rest_framework/backends.py

```diff
@@ -130,14 +153,13 @@ def get_schema_fields(self, view):
 
         filterset_class = self.get_filterset_class(view, queryset)
 
-        return [] if not filterset_class else [
-            compat.coreapi.Field(
-                name=field_name,
-                required=field.extra['required'],
-                location='query',
-                schema=self.get_coreschema_field(field)
-            ) for field_name, field in filterset_class.base_filters.items()
-        ]
+        if not filterset_class:
+            return []
+
+        return [self.build_coreapi_field(schema_field_name, field)
+                for field_name, field in filterset_class.base_filters.items()
+                for schema_field_name in self.get_schema_field_names(field_name, field)
+                ]
 
     def get_schema_operation_parameters(self, view):
```

Comment:
This is the method that generates the OpenAPI schema details.

Reply:
Hi @carltongibson that's fine I can add the change for ```get_schema_operation_parameters```. One thing to note is the change that I originally made solved the rendering field issue (must be pointing to using the old CoreAPI fields). Are you happy for me to put both of these changes under this PR?

Reply:
OK, no problem.

Reply:
@carltongibson this has now been added